index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
Não era um requisito, mas eu colocaria um título bacana na página:
```suggestion
    <title>Lições Aprendidas por John Pierre</title>
```

Reply:
Anotado! Obrigado.